fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TabItemTokens.kt

```diff
@@ -66,49 +66,53 @@ open class TabItemTokens : IControlToken, Parcelable {
     }
 
     @Composable
-    open fun iconColor(tabItemInfo: TabItemInfo): StateColor {
+    open fun iconColor(tabItemInfo: TabItemInfo): StateBrush {
         return when (tabItemInfo.fluentStyle) {
-            FluentStyle.Neutral -> StateColor(
-                rest = FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground3].value(
+            FluentStyle.Neutral -> StateBrush(
+                rest = SolidColor( FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground3].value(
                     themeMode = FluentTheme.themeMode
```

Comment:
why we need to wrap this with SolidColor?

Reply:
To cast a Color object into a Brush object, we wrap it with SolidColor